test/junit/scala/SerializationStabilityTest.scala

```diff
@@ -81,7 +81,7 @@ object SerializationStability {
     }
   }
 
-  // Generated on 20180605-18:45:47 with Scala version 2.13.0-20180604-234247-8dd6ca5)
+  // Generated on 20180701-21:01:46 with Scala version 2.13.0-20180701-205044-6e3f96b)
```

Comment:
Should we override this file?

Reply:
Why was this file overridden? That’s the first time I see it.

Reply:
The changes broke the test, so I had to update the test. Doing so automatically also updates the timestamp and hash

Reply:
I'm not actually sure *why* the changes broke the test, since theoretically they only affect factories and companion objects, which shouldn't store any serialization data

Reply:
This is safe to regenerate at the moment. All collection classes have new SerialVersionUIDs and are serialization incompatible with M3 and earlier versions anyway.